### PR TITLE
Reroute the Tweets array when in an object

### PR DESCRIPTION
**Pull request by jameswburke**

### CODE REVIEW

File: class-tapi-query.php [hunk 1]
```diff
@@ -59,6 +59,11 @@ private function _resolve_type() {
 
 	public function query() {
 		$this->tweets = WP_Twitter_API()->get( $this->path, $this->query_args );
+		if ( is_object( $this->tweets ) ) {
+			if ( ! empty( $this->tweets->statuses ) ){
+				$this->tweets = $this->tweets->statuses;
+			}
+		}
 		if ( is_array( $this->tweets ) && ! empty( $this->tweets ) && ! isset( $this->tweets->errors ) ) {
```
Comment (scottnelle) [inline class-tapi-query.php:68]: Should this be `elseif`? obviously these two conditions can't both be true so it's not a big deal, but maybe worth changing for clarity.

File: class-tapi-query.php [hunk 2]
```diff
@@ -59,6 +59,11 @@ private function _resolve_type() {
 
 	public function query() {
 		$this->tweets = WP_Twitter_API()->get( $this->path, $this->query_args );
+		if ( is_object( $this->tweets ) ) {
```
Comment (scottnelle) [inline class-tapi-query.php:63]: consider adding a comment like `// The search endpoint returns an object instead of an array of tweets` or whatever.

**Comment by scottnelle** [comment]
Couple comments. :ghost:

File: class-tapi-query.php (hunk 1 above)
Comment (jameswburke) [inline class-tapi-query.php:68]: Line 64 turns it into an array, which should then trigger true for 67

Comment (scottnelle) [inline class-tapi-query.php:68]: Ah yeah, sorry. Must've been sleepwalking.

**Pull request merged**